frontend/src/components/PostsPagination.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { usePagination } from '../utils/customHooks/usePagination'
+import PaginationElement from './PaginationElement'
+
+const PostsPagination = ({ posts, savePosts }) => {
+  const pagination = usePagination(posts)
+
+  if (!posts.data.length) {
+    return null
+  }
+
+  const handleClick = page => {
+    switch (page) {
```

Comment:
I dont like the idea of mixing types in switch, as I understand sometimes it would be a number of a page and in other cases string representing next or previous page.